### PR TITLE
Updated INSTALL.md

### PR DESCRIPTION
**Pull request by ros2torial**
Updated INSTALL.md so that IGH EtherCAT master can be installed in new system without showing errors.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -33,10 +48,20 @@ Now you can start the EtherCAT master:
...
 $ sudo /etc/init.d/ethercat start
 ```
+it should print
+```shell
+Starting EtherCAT master 1.5.2  done
+```
+
 Make sure your current user is member of `ecusers` and check connected slaves:
 ```shell
+$ sudo chmod 777 /dev/EtherCAT0
```
Comment (mcbed) [inline INSTALL.md:58]: if you are part of `ecusers` you should not need this.

File: INSTALL.md [hunk 2]
```diff
@@ -4,25 +4,40 @@
...
-```shell
-$ ./bootstrap # to create the configure script, if downloaded from the repository
-
-$ ./configure --prefix=/usr/local/etherlab  --disable-8139too --enable-generic # Ethernet driver e1000e not supported for kernels 4.X
+$ sudo apt-get install autoconf
+$ sudo apt-get install libtool
+$ sudo apt-get install pkg-config
+$ sudo apt-get install make
+$ sudo apt-get install build-essential
+$ sudo apt-get install net-tools
```
Comment (mcbed) [inline INSTALL.md:18]: maybe on a single line for better readability ?

File: INSTALL.md [hunk 3]
```diff
@@ -4,25 +4,40 @@
...
+$ sudo rm /usr/bin/ethercat
+$ sudo rm /etc/init.d/ethercat
```
Comment (mcbed) [inline INSTALL.md:20]: is there a practical reason to do this ?

**Review by mcbed — APPROVED** [review]
Thanks! Just some minor comments.

File: INSTALL.md (hunk 3 above)
Comment (ros2torial) [reply INSTALL.md:20]: yes. If already these files are there then they create issues if we want to reinstall. that's why clearing the ethercat files if already were there for smooth installation.

File: INSTALL.md (hunk 1 above)
Comment (ros2torial) [reply INSTALL.md]: okay. removed this line in the latest commit.

File: INSTALL.md (hunk 2 above)
Comment (ros2torial) [reply INSTALL.md]: okay. corrected this in the latest commit.

**Review by mcbed — APPROVED** [review]

**Pull request merged**